conda_smithy/cli.py

```diff
@@ -345,6 +376,31 @@ def __call__(self, args):
         else:
             print("Drone registration disabled.")
 
+        if args.cirun:
+            print("Cirun Registration")
+            if args.remove:
+                if args.cirun_resources:
+                    to_remove = args.cirun_resources
+                else:
+                    to_remove = ["*"]
+
+                print(f"Cirun Registration: resources to remove: {to_remove}")
+                for resource in to_remove:
+                    conda_smithy.cirun_utils.remove_repo_from_cirun_resource(
+                        owner, repo, resource
+                    )
+            else:
+                print(
+                    f"Cirun Registration: resources to add to: {owner}/{repo}"
+                )
+                conda_smithy.cirun_utils.enable_cirun_for_project(owner, repo)
+                for resource in args.cirun_resources:
```

Comment:
```suggestion
                for resource in (args.cirun_resources or ()):
```

Fix for the bug above.